backend/migrations/20230922175734-add-global-vars.sql

```diff
@@ -0,0 +1,12 @@
+-- +migrate Up
+CREATE TABLE global_vars (
+  id INT AUTO_INCREMENT,
+  name VARCHAR(255) NOT NULL UNIQUE,
+  value VARCHAR(255) NOT NULL,
```

Comment:
Is VARCHAR(255) large enough? What if* I wanted to pass an RSA key or longer document?

Reply:
oh that's a good point - yeah I can def change that

Reply:
changed the type to TEXT - how does that sound?

Reply:
just tested by adding a really long doc and the UI 'breaks' so I'll fix that as well